sematic/container_images.py

```diff
@@ -3,6 +3,7 @@
 import re
 from typing import Dict
 
+# Third-party
```

Comment:
Not third-party

Reply:
Adding it to `STDLIB` messed up all the imports, so I just deactivated it.